invenio_app_ils/ui/src/invenio_app_ils/common/api/items/serializer.js

```diff
@@ -2,26 +2,18 @@ import _isEmpty from 'lodash/isEmpty';
 import { fromISO } from '../date';
 
 export function serializeResponse(hit) {
-  const result = { ...hit.metadata };
+  let result = {};
   if (!_isEmpty(hit)) {
-    result['item_pid'] = hit.metadata.item_pid;
+    result['id'] = hit.id;
```

Comment:
This `hit.id` I think it will be an integer, https://books-dev.web.cern.ch/api/circulation/loans/?q=? 

In case of an integer we have to cast it to string, we have several of these issues everywhere :/

I am not going to add this comment on every case that follows, if I am right please have a look!

Reply:
I have put the item_pid in the relevant field, which is a string. This one keeps the elastic search structure, with only dates serialised, to be able to use them in JS.